setup.py

```diff
@@ -190,6 +191,10 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+        for name in glob.glob(os.path.join(dir_setup,"doc","src","modules","physics","vector","*.pdf")):
```

Comment:
please add spaces after commas, see pep 8. And wrap the line if it is longer than 79 characters.